src/views/Jobs/JobResultsTable.vue

```diff
@@ -780,6 +810,14 @@ export default Vue.extend({
         this.selected = [];
       } else {
         this.selected = [...this.filteredPathways];
+        if (
+          this.selected.some(
+            jobPrediction =>
+              jobPrediction.method === "PathwayPredictor+DifferentialFVA"
+          )
+        ) {
+          this.isDiffFvaChecked = true;
+        }
```

Comment:
Minor detail, but you could reuse the `onCheckboxChange` method here to avoid implementing the logic twice.

```
this.selected.forEach(jobPrediction => { this.onCheckboxChange(jobPrediction.method); });
```